futlib/sobol.fut

```diff
@@ -26,6 +26,7 @@ module type sobol = {
   module Reduce :
       (X : { include monoid
              val f : [D]f64 -> t }) -> { val run : i32 -> X.t }
+  val redomap 't : ([D]f64 -> t) -> (t -> t -> t) -> t -> i32 -> t
```

Comment:
Why is the neutral element required?  Could we not simply require that if the chunk function is applied to an empty array of `f64`s, then the result must be the neutral element?